client/scss/components/_promo.scss

```diff
@@ -28,10 +68,102 @@
   }
 }
 
-.promo__intro {
+.promo__image {
+  display: block;
+  width: 100%;
+}
+
+.promo__defs {
+  position: absolute;
+  height: 0;
+  width: 0;
+  overflow: none;
+  left: -100%;
+}
+
+.promo__image-mask {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  filter: url('#duotone') saturate(80%);
+}
+
+.promo__clip-path--1 {
+  clip-path: polygon(0% 65%, 0% 100%, 100% 100%, 100% 64%, 92% 57%, 70% 50%, 57% 60%, 45% 68%, 20% 70%);
+}
+
+.promo__clip-path--2 {
+  clip-path: polygon(0% 55%, 0% 100%, 100% 100%, 100% 60%, 82% 57%, 74% 54%, 60% 60%, 55% 63%, 40% 60%, 27% 64%);
+}
+
+.promo__clip-path--3 {
+  clip-path: polygon(0% 45%, 0% 100%, 100% 100%, 100% 42%, 75% 59%, 47% 50%, 27% 54%);
+}
+
+.promo__clip-path--4 {
+  clip-path: polygon(0% 39%, 0% 100%, 100% 100%, 100% 50%, 72% 45%, 62% 55%, 52% 44%, 48% 50%, 30% 45%, 17% 42%);
+}
+
+.promo__clip-path--5 {
+  clip-path: polygon(0% 58%, 0% 100%, 100% 100%, 100% 60%, 90% 68%, 84% 57%, 60% 60%, 45% 33%, 30% 40%, 13% 50%);
+}
+
+.promo__clip-path--6 {
+  clip-path: polygon(0% 21%, 0% 100%, 100% 100%, 100% 24%, 92% 30%, 84% 26%, 70% 37%, 45% 45%, 36% 49%, 27% 42%, 14% 28%);
+}
+
+.promo__clip-path--7 {
+  clip-path: polygon(0% 52%, 0% 100%, 100% 100%, 100% 57%, 70% 60%, 50% 53%, 40% 60%, 27% 64%);
+}
+
+.promo__clip-path--8 {
+  clip-path: polygon(0% 45%, 0% 100%, 100% 100%, 100% 45%, 82% 52%, 74% 44%, 60% 50%, 55% 57%, 40% 52%, 27% 48%);
+}
+
+.promo__clip-path--9 {
+  clip-path: polygon(0% 65%, 0% 100%, 100% 100%, 100% 30%, 84% 37%, 69% 32%, 60% 60%, 55% 63%, 40% 60%, 27% 64%);
+}
+
+.promo__clip-path--10 {
```

Comment:
I'm worried about the browser perf on all of this?
We are SVGing the hell out of the site... Any scope to get a low perf device for the team @jennpb?